interfaces/Portalicious/src/app/services/paginate-query.service.ts

```diff
@@ -100,6 +113,35 @@ export class PaginateQueryService {
     };
   }
 
+  private getDateFilterValue(date: Date, matchMode?: string): string {
+    const { startOfDay, endOfDay } = this.getStartAndEndOfDay(date);
+    switch (matchMode) {
+      case FilterMatchMode.EQUALS:
+        return `${startOfDay.toISOString()},${endOfDay.toISOString()}`;
+      case FilterMatchMode.GREATER_THAN:
+        return endOfDay.toISOString();
+      case FilterMatchMode.LESS_THAN:
+        return startOfDay.toISOString();
+      default:
+        return date.toISOString();
+    }
+  }
+
+  private getStartAndEndOfDay(date: Date): {
+    startOfDay: Date;
+    endOfDay: Date;
+  } {
+    const startOfDay = new Date(date);
+    startOfDay.setHours(0, 0, 0, 0);
+    startOfDay.setMinutes(
+      startOfDay.getMinutes() - startOfDay.getTimezoneOffset(),
+    );
+    const endOfDay = new Date(date);
+    endOfDay.setHours(23, 59, 59, 999);
+    endOfDay.setMinutes(endOfDay.getMinutes() - endOfDay.getTimezoneOffset());
```

Comment:
We're only handling UTC-timestamps, right?

So maybe we should be using the `setUTCHours`-methods, so we don't have to juggle with timezones...?

Or could be just use `T00:00:00:000Z` (as start) and `T23:59:59:999Z` (as end) after the `date`?

Reply:
On second thought...

In the `e2e/`-folder/tests, we use the package `date-fns`: https://date-fns.org/

Maybe we should use that everywhere?